packages/gatsby/package.json

```
@@ -121,7 +121,7 @@
     "v8-compile-cache": "^1.1.0",
     "webpack": "^4.12.0",
     "webpack-dev-middleware": "^3.0.1",
-    "webpack-dev-server": "^3.1.1",
+    "webpack-dev-server": "^3.1.11",
```

Might as well update to the latest

```suggestion
     "webpack-dev-server": "^3.1.14",
```